package.json

```diff
@@ -40,6 +41,7 @@
     "json-loader": "^0.5.4",
     "karma": "^0.13.22",
     "karma-chai": "^0.1.0",
+    "karma-cli": "^1.0.0",
```

Comment:
I needed to add this to make ```npm test``` work (at least in Windows), I hope you don't mind :)